lucet-runtime/lucet-runtime-internals/src/instance/signals.rs

```diff
@@ -357,13 +377,35 @@ bitflags! {
     }
 }
 
-pub unsafe fn sigaltstack(ss: &SigStack) -> nix::Result<SigStack> {
-    let mut oldstack = std::mem::uninitialized::<libc::stack_t>();
-
+pub unsafe fn sigaltstack(new_sigstack: Option<SigStack>) -> nix::Result<Option<SigStack>> {
```

Comment:
We might want to revisit this interface if we decide to upstream to `nix`, but this does the job we need it to do in this project